test/hooks.test.js

```diff
@@ -319,10 +319,10 @@ test('onRoute hook should be called / 2', t => {
     })
     next()
   })
-  .after(() => {
-    t.strictEqual(firstHandler, 1)
-    t.strictEqual(secondHandler, 1)
-  })
+    .after(() => {
```

Comment:
I still don't understand why this would be failing on Windows and nowhere else. If this must be done, I think the following would be better:

```js
instance
  .addHook()
  .after()
```

Reply:
I fixed those under mac os, standard v11 was throwing error anyway.

Reply:
I actually ran that this morning (couple hours ago) and nothing changed 😕

Reply:
With which version of standard?

Reply:
Doh! I forgot to `npm install` after switching to the branch. Did I say it was morning when I did it?